### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by soreau**
Minor changes to fix warnings.

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -39,6 +39,14 @@ struct wf_recorder_output
 
 std::vector<wf_recorder_output> available_outputs;
 
+static timespec get_ct()
```
Comment (ammen99) [inline src/main.cpp]: is this function used at all?

Comment (soreau) [reply src/main.cpp]: It's used to output some info but that line could be removed with the entire function.
`std::cout << "start at " << timespec_to_usec(get_ct()) / 1.0e6<< std::endl;`

Comment (ammen99) [reply src/main.cpp]: yeah that was a debug print, we don't need it

Comment (soreau) [reply src/main.cpp]: Ok fixed, I just merged everything into a single commit.

**Comment by ammen99** [comment]
Looks good, thanks!

**Pull request merged**